string id
sensor_msgs/Image imageBGR
---
face_recog/VisionFaceObjects faces
